### DISCUSSION TITLE
Cardboard MiniDexed Build Tutorial

### DISCUSSION THREAD

**Discussion by NickCulbertson** [probonopd/MiniDexed#640] [Show and tell]
I made a tutorial on putting together MiniDexed in a cardboard enclosure. 

Thank you to all the contributors here. This thing sounds amazing!

DIY Cardboard FM Synth (8 DX7s!) | MiniDexed & Raspberry Pi
https://youtu.be/uFpr1-5iYFI?si=uETc8GHIXvTXBsG3

**Comment by probonopd** [comment]
Who needs a 3D printer when there is cardboard ;-)
Thanks for this great video @NickCulbertson.

**Reply by NickCulbertson** [reply]
Thanks, @probonopd!

I'd love to have a 3d printer one day. I literally used the box the LCD display came in, haha! At least if I ever move, it's already packed up.

**Comment by diyelectromusic** [comment]
That's a great video.  Nice to see the MIDI buttons being used well!

And great that there is so much about the Performance voices - we can thank @Banana71 for many of those :)

Kevin

**Reply by NickCulbertson** [reply]
Thanks, Kevin!

Yeah, I completely missed those Performance Presets when I was first setting everything up. I should have taken my own advice and read the documentation. They sound so good!

**Comment by Kirtai** [comment]
Very nice example of CAD usage. (Cardboard Aided Design)

**Comment by jlam9801** [comment]
I was inspired by your build and built mine inside the box my sustain pedal came in. Do I get any points for it being a yamaha box lol

Mine is a bit more half assed than yours haha, but this was just a temporary mock up until I can build something out of wood for a permanent case

Here's a Pic of it

https://imgur.com/a/lm3GGdj

**Reply by NickCulbertson** [reply]
Haha! That's awesome! Yeah, the Yamaha box makes it legit. 🙌
